gengo/gengo.py

```diff
@@ -248,7 +249,7 @@ def get(self, **kwargs):
             # best to familiarize yourself  with the Gengo authentication
             # API. (http://developers.gengo.com/)
             query_params = dict([k, quote(str(v).encode('utf-8'))] for k, v
-                                in kwargs.items())
+                                in list(kwargs.items()))
```

Comment:
agreed with @narusemotoki that this seems unnecessary. would be nice to hear from you why this was needed! :)